kafka-ui-react-app/src/components/Connect/Details/Actions/Actions.tsx

```diff
@@ -100,7 +100,7 @@ const Actions: React.FC<ActionsProps> = ({
   }, [resumeConnector, clusterName, connectName, connectorName]);
 
   return (
-    <ConnectorActionsWrapperStyled>
+    <ConnectorActionsWrapperStyled data-testid="actions_view">
```

Comment:
This is not a "best practice" approach.

Reply:
done

Reply:
I can see a lot of them are still left in the code

Reply:
done